pynest/nest/lib/hl_api_simulation.py

```diff
@@ -188,6 +188,102 @@ def SetKernelStatus(params):
+        directory)
+    data_prefix : str
+        A common prefix for all data files
+    overwrite_files : bool
+        Whether to overwrite existing data files
+    print_time : bool
+        Whether to print progress information during the simulation
+    use_wfr : bool
+        Whether to use waveform relaxation method
+    wfr_comm_interval : double
+        Desired waveform relaxation communication interval
+    wfr_tol : double
+        Convergence tolerance of waveform relaxation method
+    wfr_max_iterations : int
+        Maximal number of iterations used for waveform relaxation
+    wfr_interpolation_order : int
+        Interpolation order of polynomial used in wfr iterations
+    dict_miss_is_error : bool
+        Whether missed dictionary entries are treated as errors
+    adaptive_spike_buffers  : bool
+        Whether MPI buffers for communication of spikes resize on the fly
+        (read/write)
+    adaptive_target_buffers : bool
+        Whether MPI buffers for communication of connections resize on the fly
+        (read/write)
+    buffer_size_spike_data : int
+        Total size of MPI buffer for communication of spikes (read/write)
+    buffer_size_target_data : int
+        Total size of MPI buffer for communication of connections (read/write)
+    growth_factor_buffer_spike_data : float
+         If MPI buffers for communication of spikes resize on the fly, grow
+         them by this factor each round (read/write)
+    growth_factor_buffer_target_data : float
+        if MPI buffers for communication of connections resize on the fly, grow
+        them by this factor each round (read/write)
+    keep_source_table : bool
+        Whether to keep source table after connection setup is complete
+        (read/write)
+    max_buffer_size_spike_data : int
+        Maximal size of MPI buffers for communication of spikes (read/write)
+    max_buffer_size_target_data : int
+        Maximal size of MPI buffers for communication of connections
+        (read/write)
+    max_num_syn_models : int
+        Maximal number of synapse models supported (read only)
+    sort_connections_by_source : bool
+        whether to sort connections by their source; increases construction
+        time of presynaptic data structures, decreases simulation time if the
+        average number of outgoing connections per neuron is smaller than the
+        total number of threads (read/write)
+    structural_plasticity_synapses : dict
+        Defines all synapses which are plastic for the structural plasticity
+        algorithm. Each entry in the dictionary is composed of a synapse model,
+        the pre synaptic element and the post synaptic element. (read/write)
+    structural_plasticity_update_interval : int
+        defines all synapses which are plastic for the structural plasticity
+        algorithm. Each entry in the dictionary is composed of a synapse model,
+        the pre synaptic element and the post synaptic element. (read/write)
```

Comment:
Shouldn't this be ``Defines the time interval in ms at which the structural plasticity manager will make changes in the structure of the network (creation and deletion of plastic synapses).``?

Reply:
Thanks for spotting this one! Yes indeed